marimo/_islands/island_generator.py

```diff
@@ -249,3 +282,17 @@ def render_head(
 
 def remove_empty_lines(text: str) -> str:
     return "\n".join([line for line in text.split("\n") if line.strip() != ""])
+
+
+def handle_mimetypes(output: CellOutput) -> str:
```

Comment:
you might be able to do `marimo.as_html(output.data).text`

Reply:
Didn't handle images, and HTML was outputted literally- but I do this there's probably a better way